content/docs/user-guide/interactivity/physics/debugging.md

```diff
@@ -87,3 +88,15 @@ You must enable the [ImGui Gem](/docs/user-guide/gems/reference/debug/imgui) to
 ## Debug Options in the PhysX Configuration 
 
 You can also specify debug settings in the **PhysX Configuration** tool. See [Debugger Configuration](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-debugger/).
+
+## Enable additional checks and error reporting in PhysX SDK
+
+You can make the profile configuration of O3DE to use the **checked** version of PhysX SDK library. PhysX will perform additional checks to detect invalid parameters, API race conditions and other incorrect uses of the API which might otherwise cause mysterious crashes or failures in simulation. The benefit is to enable the same safety checks as in debug configuration but without having to run O3DE on debug, where low framerates could spoil the simulation.
```

Comment:
Extreme nitpick - I'd suggest changing "make the profile configuration of O3DE to use" to "make the profile configuration of O3DE use" i.e. dropping the "to".

Reply:
Thanks, done.